Frontend/library/src/DataChannel/DataChannelLatencyTestController.ts

```diff
@@ -67,26 +70,35 @@ export class DataChannelLatencyTestController {
-                return acc + (next.playerReceivedTimestamp - next.playerSentTimestamp);
-            }, 0) / this.records.size),
-            playerToStreamerTime: Math.ceil(Array.from(this.records.values()).reduce((acc, next) => {
-                return acc + (next.streamerReceivedTimestamp - next.playerSentTimestamp);
-            }, 0) / this.records.size),
-            streamerToPlayerTime: Math.ceil(Array.from(this.records.values()).reduce((acc, next) => {
-                return acc + (next.playerReceivedTimestamp - next.streamerSentTimestamp);
-            }, 0) / this.records.size),
-            exportLatencyAsCSV: () => {
-                let csv = "Timestamp;RTT;PlayerToStreamer;StreamerToPlayer;\n";
-                resultRecords.forEach((record) => {
-                    csv += record.playerSentTimestamp + ";";
-                    csv += (record.playerReceivedTimestamp - record.playerSentTimestamp) + ";";
-                    csv += (record.streamerReceivedTimestamp - record.playerSentTimestamp) + ";";
-                    csv += (record.playerReceivedTimestamp - record.streamerSentTimestamp) + ";";
-                    csv += "\n";
-                })
-                return csv;
-            }
+                dataChannelRtt: Math.ceil(
+                    Array.from(this.records.values())
+                        .reduce(
+                            (acc, next) => {
+                                return acc + (next.playerReceivedTimestamp - next.playerSentTimestamp);
+                            },
+                            0) /
+                    this.records.size),
+                playerToStreamerTime: Math.ceil(
+                    Array.from(this.records.values())
+                        .reduce(
+                            (acc, next) => {
+                                return acc + (next.streamerReceivedTimestamp - next.playerSentTimestamp);
+                            },
+                            0) /
+                    this.records.size),
+                streamerToPlayerTime: Math.ceil(Array.from(this.records.values()).reduce((acc, next) => {
+                    return acc + (next.playerReceivedTimestamp - next.streamerSentTimestamp);
+                }, 0) / this.records.size), exportLatencyAsCSV: () => {
+                    let csv = 'Timestamp;RTT;PlayerToStreamer;StreamerToPlayer;\n';
+                    resultRecords.forEach((record) => {
+                        csv += record.playerSentTimestamp + ';';
+                        csv += (record.playerReceivedTimestamp - record.playerSentTimestamp) + ';';
+                        csv += (record.streamerReceivedTimestamp - record.playerSentTimestamp) + ';';
+                        csv += (record.playerReceivedTimestamp - record.streamerSentTimestamp) + ';';
+                        csv += '\n';
+                    })
+                    return csv;
+                }
```

Comment:
How you had it before the formatter seemed better.

Reply:
I'm not sure why it's decided to add an additional intent here.